cloud-db/acl-overview/android.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Access Control Overview
+---
+
+[[toc]]
+
+Skygear has provided a default security setting on all records.
+
+By default,
+
+- read access is granted to the public, and
+- write access is granted only to owner of the record.
+
+Read access includes access to **query** and **fetch**. Write access includes access to **update** and **delete**. More details below.
+
+For example:
+
+```java
+// not logged in as any user
+Query noteQuery = new Query("note");
+Database publicDB = skygear.getPublicDatabase();
+
+publicDB.query(noteQuery, new RecordQueryResponseHandler() {
+    @Override
+    public void onQuerySuccess(Record[] records) {
+        // Successful to query Note records
+        Log.i("ACL", String.format("Successfully got %d records", records.length));
+        Record aNote = records[0];
+        aNote.set("content", "Hello World!");
+        publicDB.save(aNote, new RecordSaveResponseHandler(){
+            @Override
+            public void onSaveSuccess(Record[] records) {
+            }
+
+            @Override
+            public void onPartiallySaveSuccess(
+                Map<String, Record> successRecords,
+                Map<String, String> reasons
+            ) {
+            }
+
+            @Override
+            public void onSaveFail(String reason) {
+              // Fail for not authenticated user
+              Log.i("ACL", String.format("Fail with reason: %s", reason));
+            }
+        });
+    }
+
+    @Override
+    public void onQueryError(String reason) {
+    }
+});
+```
+
+If you want to customize these security settings, Skygear supports access control on records. You can use it to control whether a user can query, update or delete a record.
+
+Skygear uses [Access Control List][wiki-acl] (ACL) as the model of access control.
+An ACL is a list of Access Control Entry (ACE), each ACE describes which
+access rights a user target has to a record.
+
+Each ACE consists of:
+- **Access level**, which defines what actions can be performed (for example, read or write).
+- **User Target**, which defines who can perform the specified actions (for example, a specific user or group of users).
```

Comment:
Each ACE consists of:

**Resource**
**User target**
**Access Level**

Missing Resource here.